app/api/submission/Representative.scala

```diff
@@ -26,8 +26,10 @@ object Representative {
       r =>
         RepresentativeType05(
           r.eori.value,
-          r.capacity.code,
-          Some(ContactPersonType05(r.name, r.phone, None))
+          "direct",
```

Comment:
I think the code for direct is `"2"` judging by this - https://github.com/hmrc/manage-transit-movements-departure-frontend/blob/11dad387b93264be6ccf4bf41e9ffb5020623f36/app/models/traderDetails/representative/RepresentativeCapacity.scala#L28

Reply:
Yea you're correct, it should be using the code that was associated with the object - pushed that